sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -933,6 +933,21 @@ class DataSourceV2SQLSuite
     }
   }
 
+  test("AlterNamespaceSetProperties using v2 catalog") {
+    withNamespace("testcat.ns1.ns2") {
+      sql("CREATE NAMESPACE IF NOT EXISTS testcat.ns1.ns2 COMMENT " +
+        "'test namespace' LOCATION '/tmp/ns_test' WITH PROPERTIES ('a'='a','b'='b','c'='c')")
+      sql("ALTER NAMESPACE testcat.ns1.ns2 SET PROPERTIES ('a'='b','b'='a','c'='c')")
```

Comment:
sorry I didn't make it clear. I mean "untouched", so the query can be `SET PROPERTIES ('a'='b','b'='a')`

Reply:
ok, got it.

Reply:
updated